translate/snippets/snippet.go

```diff
@@ -1,147 +0,0 @@
-// Copyright 2019 Google LLC
```

Comment:
Sorry I missed this - we have to make sure docs can always be republished. If we delete this file, the existing references in docs to this file will break. So, we either need to:
* Tag the docs to use a particular branch, submit this PR, update the doc references, and remove the branch specification, or
* Leave this file here (being careful with function names and stuff), submit the PR, update all of the references, then submit a second PR to remove this file.

I'd go with the second option and move all of the snippets one level up (outside the `snippets` directory) to avoid naming collisions.

Reply:
Done